components/storage/blockdevice/COMPONENT_QSPIF/QSPIFBlockDevice.cpp

```diff
@@ -247,7 +247,7 @@ int QSPIFBlockDevice::init()
 
     // Configure  BUS Mode to 1_1_1 for all commands other than Read
     _qspi_configure_format(QSPI_CFG_BUS_SINGLE, QSPI_CFG_BUS_SINGLE, QSPI_CFG_ADDR_SIZE_24, QSPI_CFG_BUS_SINGLE,
-                           QSPI_CFG_ALT_SIZE_8, QSPI_CFG_BUS_SINGLE, 0);
+                           0, QSPI_CFG_BUS_SINGLE, 0);
```

Comment:
I don't understand this change. Documentation says:
```
*  @param alt_size Size in bits used by alt phase(Valid values are QSPI_CFG_ALT_SIZE_8, QSPI_CFG_ALT_SIZE_16, QSPI_CFG_ALT_SIZE_24, QSPI_CFG_ALT_SIZE_32)
```

So why zero instead of one of the valid values?

Reply:
Sorry, you changed it to:
```
*  @param alt_size Size in bits used by alt phase (must be a multiple of the number of bus lines indicated in alt_width)
```

But I still don't get it, why zero is multiple of bus lines?

Reply:
Prior to this change, QSPI_CFG_ALT_SIZE_8 seems to be chosen as a somewhat arbitrary placeholder (the code has no particular knowledge that 8 alt bits are actually what is required) which then winds up being ignored because QSPIFBlockDevice never passes anything other than -1 for alt. 
Setting 0 (no alt bits) seems like a more direct way of expressing what QSPIFBlockDevice is doing. However, I'm okay reverting this to 8 for now if you'd prefer given the more significant rework of alt bit handling in #11531 .

Reply:
No need to revert. Explanation is enough. Zero is OK to express "No alt bits"